Auth0/Auth0.manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "author": "Auth0",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "Auth0 is a cloud-based, extensible identity provider for various applications. The Sumo Logic App for Auth0 makes it easy to analyze and visualize your Auth0 logs, and provides insight into security and operational issues.",
+    "helpURL": "https://auth0.com/docs/logs/streams/sumo-logic-dashboard",
+    "hoverText": null,
+    "iconURL": "https://app_icons.s3.amazonaws.com/auth0.png",
```

Comment:
@sumo-drew oops, was I supposed to leave the `iconURL` value alone?

Reply:
I don't think it matters as we set this during build and deploy. @himanshu219 can confirm.

Reply:
yes it doesn't matter.

Reply:
Okay @sumo-apps-team - if we fix this really fast, is there still time to deploy by Monday?

Reply:
@sumo-drew @sumo-apps-team Can I just delete these last two commits then and we can get this merged?

Reply:
which two commits? actually when the test will run they automatically replace these urls and you will see a commit in your PR with description manifest updates.